zng/cc.rs

```diff
@@ -238,7 +246,7 @@ pub fn build_zlib_ng(target: &str, compat: bool) {
 
             // SSE4.2
             cfg.define("X86_SSE42", None);
-            cfg.append(Some("arch/x86"), &["adler32_sse42", "insert_string_sse42"]);
```

Comment:
What's the background to removing this?

Reply:
`insert_string_sse42` doesn't exists anymore.